File: Practice/src/exercise/SplitString.js
function solution(s) {
    var answer = 0;
    let count = 0;
    for (let i = 0, j = 0; i < s.length; i++) {
        if (i == j) {
            count = 1;
            answer++;
        } else if (s[i] == s[j]) {
            count++;
        } else {
            count--;
        }
        if (count == 0) {
            j = i + 1;
        }
    }
    return answer;
}

File: Practice/src/exercise/SplitString.py
def solution(s):
    answer = 0
    j = ""
    count = 0
    for i in s:
        if not count:
            answer += 1
            j = i;
            count = 1
        elif i == j:
            count += 1
        else:
            count -= 1
    return answer
